gprofiler/profilers/java.py

```diff
@@ -503,6 +509,11 @@ class JavaProfiler(ProcessProfilerBase):
         17: (Version("17.0.1"), 12),
     }
 
+    # extra timeout seconds to add to the duration itself.
+    # once the timeout triggers, AP remains stopped, so if it triggers before we tried to stop
+    # AP ourselves, we'll be in messed up state. hence, we add 30s which is enough.
```

Comment:
Because we think AP is running and that our "stop" request will stop it, but it's already stopped so a "stop" request won't do what we expect (i.e I think it won't write a output file, which we'll later try to read; but I didn't verify that)